src/ClientProtos/capture_data.proto

```diff
@@ -175,12 +175,16 @@ message TimerInfo {
 }
 
 message ApiStringEvent {
-  uint32 process_id = 1;
```

Comment:
I would keep at least the `proccess_id` field so that multi-process profiling remains possible.

Reply:
These "client protos" are not part of the capture format. We use them for legacy reasons but they could and should be structs. We can just as easily add fields back when we need them.